src/components/Breadcrumbs/Breadcrumbs.test.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import Breadcrumbs from './index';
+import { renderWithTheme } from '../../../jest/utils';
+
+const mockCrumbs = [
+  {
+    children: 'First page',
+    href: '/',
+  },
+  {
+    children: 'Second page',
+    href: '/secondpage',
+  },
+  {
+    children: 'Third page',
+    href: '/thirdpage',
+  },
+  {
+    children: 'Fourth page',
+    href: '/fourthpage',
+  },
+  {
+    children: 'Fifth page',
+    href: '/fifthpage',
+  },
+  {
+    children: 'Sixth page',
+    href: '/sixthpage',
+  },
+  {
+    children: 'Seventh page',
+    href: '/seventhpage',
+  },
+];
+const truncatedCrumbs = mockCrumbs.slice(1, -2);
+const visibleCrumbs = mockCrumbs.filter(index => {
+  return truncatedCrumbs.indexOf(index) === -1;
+});
+
+describe('Breadcrumbs', () => {
+  it('renders breadcrumbs without truncation', () => {
+    const { queryAllByRole, queryByLabelText, queryByText } = renderWithTheme(
+      <Breadcrumbs items={mockCrumbs} />
+    );
+
+    // verify expected breadcrumbs and chevrons exist
+    mockCrumbs.forEach(crumb => {
+      expect(queryByText(crumb.children)).toBeTruthy();
+    });
+    expect(queryAllByRole('presentation')).toHaveLength(mockCrumbs.length - 1);
+
+    // verify truncated menu does not exist
+    expect(queryByLabelText('Truncated breadcrumb list toggle')).toBeNull();
+  });
+
+  it('renders breadcrumbs with truncation', () => {
```

Comment:
Can we extend this test to make sure that the additional breadcrumbs are displayed when you click once and hidden when you click again?